pkg/kanister.go

```diff
@@ -58,6 +59,18 @@ func Register(f Func) error {
 	return nil
 }
 
+func RegisteredFunctions() map[string]struct{} {
```

Comment:
Any reasons why we need to return a map, and not just a string slice?

Reply:
I wanted to mimic, `set` data structure that is the reason I am using map here.